File: project_euler/1/1.c
#include <stdio.h>

int main(void)
{
    const int a = 3;
    const int b = 5;
    const int upper_num = 1000;
    int sum = 0;

    for (int i = 1; i*a < upper_num; i++) {
        sum += i * a;
    }
    for (int j = 1; j*b < upper_num; ++j) {
        sum += j*b;
    }

    printf("sum is: %d\n", sum);

    return 0;
}

File: project_euler/1/1.py
#!/usr/bin/env python3
# encoding: utf-8

a = 3
b = 5
upper_num = 1000
sum_num = 0

for i in range(1, int(upper_num/a)+1):
    if i*a != upper_num:
        sum_num += i*a
for i in range(1, int(upper_num/b)+1):
    if i*b != upper_num:
        sum_num += i*b

print("sum is: ", sum_num)


